app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3 or command[:3] != "cp ":
+        return 0
+    name_f_in = command.split()[1]
+    name_f_out = command.split()[2]
```

Comment:
U can use tuple unpacking again to assign the values of name_f_in and name_f_out without calling split() twice. 
If you want to declare these variables like this, it will help to optimize the code

Reply:
Yes. It is possible. But this decision is the same as mine.
No matter, look for a new tuple or given initial string.
And in my case, I don't occupy the memory of the new variable.

Reply:
Ok, but using variable, variable, variable = command.split() and command.startswith can make your code better

Reply:
@NazarHladaniuk is right, no need to call split method for 3 times, also it is hard to read ur code
for better code style u can use split for unpack ur splitted string to variables f.e.
`arg1, arg2, arg3 = ur_string.split()`
also try to avoid using return 0 in ur code, u can just make a one backward condition to copy ur file

Reply:
`arg1, arg2, arg3 = ur_string.split()`
Now I understand what @NazarHladaniuk offered me, yesterday word 'tuple' confused me.

All comments are taken into account.